Tasks/google-play-release/task.json

```diff
@@ -1,6 +1,6 @@
 {
     "id": "8cf7cac0-620b-11e5-b4cf-8565e60f4d27",
-    "name": "google-play-release",
+    "name": "GooglePlayRelease",
```

Comment:
So we should decide which way we are going with this because the extension validation doesn't care about the dashes in the name, but if you try to uploading using the tfxcli, it complains about non alpha numeric. Given that we haven't had issues so far, and Eduardo was able to get data using the dashed names, it doesn't seem like it's a technical issue and more that the task validation is just incorrect.

What are your opinions on this? I'm leaning towards renaming then all for the sake of ease of testing.